mac

```diff
@@ -46,6 +46,16 @@ fi
 # shellcheck disable=SC2016
 append_to_zshrc 'export PATH="$HOME/.bin:$PATH"'
 
+if [ -d "/usr/local" ]; then
+  if ! sudo -u "$(whoami)" [ -r /usr/local ]; then
```

Comment:
@mike-burns We ( @gabebw and I ) decided to keep the `-r /usr/local` check to improve the script's speed.